guidelines/sc/21/reflow.html

```diff
@@ -13,9 +13,9 @@ <h4>Reflow</h4>
    </ul>
    <p>Except for parts of the content which require two-dimensional layout for usage or meaning.</p>
   
-   <p class="note">320 CSS pixels is equivalent to a starting viewport width of 1280 CSS pixels wide at 400% zoom. For web content which are designed to scroll horizontally (e.g. with vertical text), the 256 CSS pixels is equivalent to a starting viewport height of 1024px at 400% zoom.</p>
+   <p class="note">320 CSS pixels is equivalent to a starting viewport width of 1280 CSS pixels wide at 400% zoom. For web content which is designed to scroll horizontally (e.g., with vertical text), 256 CSS pixels is equivalent to a starting viewport height of 1024px at 400% zoom.</p>
```

Comment:
Yikes.

Reply:
Oh dear, I wonder if that got changed from "web pages which are designed" and not updated the is/are.